openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -320,11 +320,13 @@ This specification defines the following parameters for the Credential Offer obj
 The following values are defined by this specification: 
 
 * Grant Type `authorization_code`:
-  * `issuer_state`: OPTIONAL. String value created by the Credential Issuer and opaque to the Wallet that is used to bind the subsequent Authorization Request with the Credential Issuer to a context set up during previous steps. If the Wallet decides to use the Authorization Code Flow and received a value for this parameter, it MUST include it in the subsequent Authorization Request to the Credential Issuer as the `issuer_state` parameter value. 
+  * `issuer_state`: OPTIONAL. String value created by the Credential Issuer and opaque to the Wallet that is used to bind the subsequent Authorization Request with the Credential Issuer to a context set up during previous steps. If the Wallet decides to use the Authorization Code Flow and received a value for this parameter, it MUST include it in the subsequent Authorization Request to the Credential Issuer as the `issuer_state` parameter value.
+  * `authorization_server`: OPTIONAL when a Credential Issuer is relying on multiple Authorization Servers. MUST NOT be used otherwise. String that the Wallet can use to identify the Authorization Server to used with this grant type when `authorization_servers` parameter in the Credential Issuer metadata has multiple entries. MUST match with one of the values in the `authorization_servers` array obtained from the Credential Issuer metadata.
```

Comment:
```suggestion
  * `authorization_server`s: OPTIONAL when a Credential Issuer is relying on multiple Authorization Servers. MUST NOT be used otherwise. String that the Wallet can use to identify the Authorization Server to used with this grant type when `authorization_servers` parameter in the Credential Issuer metadata has multiple entries. MUST match with one of the values in the `authorization_servers` array obtained from the Credential Issuer metadata.
```